cspell.json

```diff
@@ -0,0 +1,18 @@
+{
+  "$schema": "https://raw.githubusercontent.com/streetsidesoftware/cspell/main/cspell.schema.json",
+  "words": [
+    "Apereo",
+    "Cacheable",
+    "ClassPath",
+    "Gradle",
+    "Jasig",
+    "JDBC",
+    "Portlet",
+    "Portlets",
+    "RDBM",
+    "Respondr",
+    "Servlet",
+    "SpringFramework",
+    "uPortal"
+  ]
+}
```

Comment:
This provides configuration for a number of integrations:
* VScode plugin https://github.com/streetsidesoftware/vscode-spell-checker
* Intellij Extension https://github.com/BlackHole1/idea-spell-check
* GitHub action https://github.com/streetsidesoftware/cspell-action
* CLI https://github.com/streetsidesoftware/cspell-cli